### PR TITLE
improved Consistency and UI-Fixes

### PR DESCRIPTION
**Pull request by PavaVo**
Improved consistency on Dutch word use and some minor word-changes to prohibit strange hyphenation on button-labels.

### CODE REVIEW

File: Localizations/nl.lproj/Localizable.strings [hunk 1]
```diff
@@ -353,18 +353,18 @@
...
+"Never confirm widget actions, even irreversible ones and actions that can accidentally grant others access to your vehicle" = "Bevestig nooit widgetacties, zelfs niet onomkeerbare acties en acties die anderen per ongeluk toegang tot je auto kunnen geven"; //Machine translated
 "Wait For Wake" = "Wacht op ontwaken";
-"Vehicle not online, keep trying?" = "Voertuig niet online, blijven proberen?"; //Machine translated
+"Vehicle not online, keep trying?" = "Auto niet online, blijven proberen?";
 "Is Driving" = "Is aan het rijden"; //Machine translated
 "Show advanced" = "Toon geavanceerd"; //Machine translated
 "Hide advanced" = "Geavanceerd verbergen"; //Machine translated
-"Calculate charger power (kW) without neutral connection" = "Bereken het laadvermogen (kW) zonder nulleider"; //Machine translated
+"Calculate charger power (kW) without neutral connection" = "Bereken het laadvermogen (kW) zonder nulleider"; //Machine translated but needs clarification
```
Comment (Kemmey) [inline Localizations/nl.lproj/Localizable.strings:368]: This is for an advanced toggle in settings where users can override how the charge power is calculated when the car is charging. Normally you would connect the tesla charger to neutral - but tesla allows the charger to be connected in a non-standard non-compatible fashion without a neutral connection. In this case, the charger power is less, and is calculated differently. There is no way to detect this weird connection in the api response, so had to be a toggle in settings.

File: Localizations/nl.lproj/Localizable.strings [hunk 2]
```diff
@@ -57,8 +57,8 @@
 "Unlock port" = "Laadpoort openen";
 "%@ port" = "Laadpoort %@";
 "Homelink" = "Homelink"; //No Machine Translation
-"Disable Sentry" = "Sentrymode Uit";
-"Enable Sentry" = "Sentrymode Aan";
+"Disable Sentry" = "Sentry Uit"; // this needs to be really short to fit in buttonbar
```
Comment (Kemmey) [inline Localizations/nl.lproj/Localizable.strings:60]: I'll have a look in the app - I would prefer localizations to not have to take rendering into account, so if there are rendering issues, I'd rather fix them generically so all languages are fixed - and not depend on sub-optimal translations.

File: Localizations/nl.lproj/Localizable.strings [hunk 3]
```diff
@@ -128,7 +128,7 @@
 "is open" = "is geopend"; //Machine translated
 "Sentry active" = "Sentry actief";
 "sentry on" = "sentry aan";
-"Seat heaters" = "Stoel verwarming";
+"Seat heaters" = "Verwarm Stoelen"; //this needs to be really short to fit on the buttonbar.
```
Comment (Kemmey) [inline Localizations/nl.lproj/Localizable.strings:131]: Same comment as for sentry

**Review by Kemmey — COMMENTED** [review]
Do you want me to merge these changes now? I'll have a look at the button label renderer and try to fix long title rendering for all languages.

File: Localizations/nl.lproj/Localizable.strings (hunk 1 above)
Comment (PavaVo) [reply Localizations/nl.lproj/Localizable.strings:368]: You can do the merges if you wish. If you change/fix the litle rendering issues I'll change it back. Now 2 words are hyphenated by one character and that's a bit awkward.

File: Localizations/nl.lproj/Localizable.strings (hunk 2 above)
Comment (PavaVo) [reply Localizations/nl.lproj/Localizable.strings:60]: Thanks, that clarifies a lot. You can delete this change (or I'll fix it in the next pull)

File: Localizations/nl.lproj/Localizable.strings (hunk 3 above)
Comment (PavaVo) [reply Localizations/nl.lproj/Localizable.strings:131]: Agreed.

**Pull request merged**